extractor/contents.py

```diff
@@ -0,0 +1,99 @@
+class Contents:
+    def __init__(self):
+        self.content_list = list()
+
+    def add_content(self, _content):
+        self.content_list.append(_content)
+
+    def print_contents(self):  # Rough printing method for testing purpose
```

Comment:
Convert this to `to_string(self)` or `debug_string(self)` and return entire debuggable string.

Reply:
When you invoke `print(contents_list.debug_string())` instead of `contents_list.print_contents()`